ui/docs/people.mdx

```diff
@@ -7,7 +7,7 @@ import People from 'taskcluster-ui/views/Documentation/People';
 # People of Taskcluster
 
 Thanks goes to these wonderful people for their hard work on Taskcluster.
-Your name [could be here, too](https://github.com/taskcluster/taskcluster/blob/master/CONTRIBUTING.md)!
+Your name [could be here, too](https://github.com/taskcluster/taskcluster/blob/master/CONTRIBUTING.mdx)!
```

Comment:
This file did not get renamed (nor should)

Reply:
Done

Reply:
Hm, did you forget to push the commit by any chance? :)

Reply:
> Hm, did you forget to push the commit by any chance? :)

I haven't pushed the changes. Waiting for reply for the second comment. Will be pushing both the fixes together.